src/git/github.js

```diff
@@ -42,12 +51,31 @@ module.exports = class Github extends Git {
 
   getFileUrl(file) {
     let url;
-    if (parsePath(file.name).ext == this.fileExt || file.name == this.filename || this.fileExt == '') { 
-      if (file.download_url) {
-        url = file.download_url;
+    if (this.release) {
+      if (parsePath(file.name).ext == this.fileExt || file.name == this.filename) {
+        url = `${this.enterprise}/repos/${this.repo}/releases/assets/${file.id}`;
+      }
+    } else {
+      if (parsePath(file.name).ext == this.fileExt || file.name == this.filename || this.fileExt == '') { 
+        if (file.download_url) {
+          url = file.download_url;
+        }
       }
     }
 
     return url;
   }
+
+  getFileArray(files) {
+    if (this.release) {
+      return files.assets;
+    } else {
+      return files;
+    }
+  }
+
+  getAddlHeaders(reqOpt) {
+    reqOpt.headers = { ...reqOpt.headers, 'Accept': 'application/octet-stream' };
```

Comment:
I believe this would override the `Accept` header if the user had already specified one. Is this the desired behavior? If so, should we warn them?

Reply:
added note to gitops doc